<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_CGerais</class>
 <widget class="QWidget" name="Dialog_CGerais">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>582</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Região de Análise</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dimenção X:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="dim_regiao_livre_x">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dimenção Y:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="dim_regiao_livre_y">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_3">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Dimenção Z:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QDoubleSpinBox" name="dim_regiao_livre_z">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix">
         <string> m</string>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="8">
       <widget class="Line" name="line_7">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>delt:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="delta_regiao_livre">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="value">
         <double>0.005000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>582</width>
       <height>45</height>
      </size>
     </property>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>160</width>
        <height>26</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_CGerais</receiver>
   <slot>slotRejected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_CGerais</receiver>
   <slot>slotAccepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim_regiao_livre_x</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>Dialog_CGerais</receiver>
   <slot>slotSetValoresAmbiente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim_regiao_livre_y</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>Dialog_CGerais</receiver>
   <slot>slotSetValoresAmbiente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dim_regiao_livre_z</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>Dialog_CGerais</receiver>
   <slot>slotSetValoresAmbiente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delta_regiao_livre</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>Dialog_CGerais</receiver>
   <slot>slotSetValoresAmbiente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotRejected()</slot>
  <slot>slotAccepted()</slot>
  <slot>slotSetValoresAmbiente()</slot>
 </slots>
</ui>
